modules/renter/contractor/contracts.go

```diff
@@ -28,6 +28,15 @@ func (c *Contractor) contractEndHeight() types.BlockHeight {
 	return c.currentPeriod + c.allowance.Period
 }
 
+// ContractUtility returns the ContractUtility for a contract with a given id.
+func (c *Contractor) contractUtility(id types.FileContractID) modules.ContractUtility {
```

Comment:
the return value here is inconsistent with `ContractUtility`. Either both should return a boolean, or neither. I prefer to return the boolean. Then `ContractUtility` can just call `contractUtility`